include/utils.php

```diff
@@ -293,8 +293,7 @@ function datetimeFormatDuration($seconds, $nbsp = false, $skipSeconds = false) {
 		// are 0 and we still want to print seconds)
 		if ($any || $qty[$i] > 0 || $i == $count - 1) {
 			if ($any) $text .= $nbsp ? '&nbsp;' : ' ';
-			if ($any && $qty[$i] < 10) $text .= '0';
-			$text .= $qty[$i].$names[$i];
+			$text .= $qty[$i].' '.$names[$i];
```

Comment:
If anyone has the same problems thinking of which difference this space makes, the following is from some "last changed" view:

> Revision 363 — 252t 23h
> Revision 363 — 252 t 23 h
> Revision 363 — 252days 23hours
> Revision 363 — 252 days 23 hours

I tend to prefer to keep no space for a single letter vs. one space like is for more than a letter. :-) In doubt I'll accept as is.

Reply:
Actually, unit style guides mandate to separate the number from the unit with a space for readability.